flow/scripts/floorplan_to_place.tcl

```diff
@@ -1,5 +1,6 @@
 # Enable keeping variables between stages
 set ::env(KEEP_VARS) 1
+set ::env(WRITE_ODB_AND_SDC_EACH_STAGE) 0
```

Comment:
This will prevent the user from overriding it.  I think this should only be set if currently unset.

Reply:
The intent of the script is to perform all operations in a single invocation.  Whether to write odb at each stage or not is a separate decision.  @eder-matheus does this matter for you?

Reply:
In the scenario I'm using it, I'm interested on the ODB files of each stage. However, I'm not against having it as an option, and the default behavior would be not writing the ODBs of each stage.